src/QirRuntime/samples/StandaloneInputReference/qir-driver.cpp

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <cstring> // for memcpy
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+#include "CoreTypes.hpp"
+#include "QirContext.hpp"
+#include "QirTypes.hpp"
+#include "QuantumApi_I.hpp"
+#include "SimFactory.hpp"
+
+#include "quantum__qis_internal.hpp"
+#include "quantum__rt.hpp"
+
+#include "CLI11.hpp"
```

Comment:
Consider placing this \#include between the <> includes and "" includes (between `#include <vector>` and `#include "CoreTypes.hpp"`). That is if we are ordering the headers from the most general to the least general.

Reply:
Changed it to the placing that you suggested.